src/harness/certs/README.md

```diff
@@ -18,11 +18,12 @@ https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/
           sas_ca--------              cbsd_ca   			   proxy_ca
           /    \        \                \              	  		\
          /      \        \                \              	  		 \
-   admin_client  server  sas     client|device_[a|c]|corrupted_client        domain_proxy
-		          |       wrong_type_client|client_expired           corrupted_domain_proxy
-                   corrupted_sas  client_inapplicable                        wrong_type_domain_proxy
-                   sas_expired                                               domain_proxy_expired
-                                                                             domain_proxy_inapplicable
+   admin_client  server  sas     client|device_[a|c]|corrupted_client        domain_proxy|
+		          |       wrong_type_client|client_expired           corrupted_domain_proxy|
+                   corrupted_sas|  client_inapplicable|                    wrong_type_domain_proxy|
+                   sas_expired|    blacklisted_client                      domain_proxy_expired|
+                   blacklisted_client                                   domain_proxy_inapplicable|
```

Comment:
I think you have a copy-and-paste error with `blacklisted_client`.

Reply:
Updated   README.md with proper blacklisted certificates information.